File: Algoritmos Traduzidos/Lua/Algoritmos(FilipeFarinha)/ex5.lua
--EXERCICIO 5: Programa que verifica se um ano é comum ou bissexto.

io.write'Digite um ano:'

ano = io.read()

if ano%400==0 then

	print'Ano Bissexto'

elseif ano%4==0 and ano%100 ~=0 then

	print'Ano Bissexto'

else

	print'Ano Comum'

end

File: Algoritmos Traduzidos/Python/Algoritmos traduzidos[Filipe]/ex5.py
ano = input('Digite um ano: ')
if int(Ano)%400 == 0:
    print('Ano Bissexto')
elif int(ano)%4 == 0 and int(ano)%100 != 0:
    print('Ano Bissexto')
else:
    print('Ano Comum')
